app/components/project-category-item.js

```diff
@@ -1,13 +1,59 @@
 import Ember from 'ember';
 
+/**
+  `project-category-item` represents a category with a project. When selected
+  the user is taken to a list of projects using this category.
+
+  ## Default usage
+  ```Handlebars
+  {{project-category-item category=category}}
+  ```
+
+  @class project-category-item
+  @module Components
+  @extends Ember.Component
+ */
 export default Ember.Component.extend({
   tagName: ['li'],
 
+  /**
+    Returns true if 'userCategory' is not empty
+
+    @property selected
+    @type Boolean
+   */
   selected: Ember.computed.notEmpty('userCategory'),
+
+  /**
+    A service that returns all userCategories.
+
+    @property userCategories
+    @type Ember.Service
+   */
   userCategories: Ember.inject.service(),
+
+  /**
+    The current user.
+
+    @property user
+    @type Ember.Object
+   */
   user: Ember.computed.alias('currentUser.user'),
+
+  /**
+    Categories that the user belongs to.
+
+    @property usersUserCategories
+    @type Ember.Array
+   */
   usersUserCategories: Ember.computed.alias('user.userCategories'),
 
+  /**
+    Returns the category.
```

Comment:
you might want to elaborate on this - I think it only returns the category if the category is in the user's categories.